components/StatsBox.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="stats-box mr-20 shadow-lg">
```

Comment:
I recommend that all components have an id or unique classname in their top element.